xml/System.Security.Authentication.ExtendedProtection/TokenBinding.xml

```diff
@@ -20,7 +20,7 @@
       <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- This class implements Token Binding using native Windows APIs. For more information, see [CNG Token Binding functions](https://msdn.microsoft.com/library/windows/desktop/mt269464\(v=vs.85\).aspx).  
+ This class implements Token Binding using native Windows APIs. For more information, see [CNG Token Binding functions](/windows/win32/seccng/cng-token-binding-functions\(v=vs.85\).aspx).  
```

Comment:
The script needs to take into account these versions numbers in the URL, otherwise you introduce errors
```suggestion

This class implements Token Binding using native Windows APIs. For more information, see [CNG Token Binding functions](/windows/win32/seccng/cng-token-binding-functions).  
```